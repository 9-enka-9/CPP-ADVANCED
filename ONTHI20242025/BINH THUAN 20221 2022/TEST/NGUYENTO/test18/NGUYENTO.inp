24311092474384