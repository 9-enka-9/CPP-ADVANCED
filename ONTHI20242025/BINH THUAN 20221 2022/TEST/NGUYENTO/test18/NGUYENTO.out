24311122058161